src/components/Provider.tsx

```diff
@@ -1,10 +1,10 @@
-import React, { useMemo } from 'react'
+import React, { ReactNode, useMemo } from 'react'
 import PropTypes from 'prop-types'
-import { ReactReduxContext } from './Context'
+import { ReactReduxContext, ReduxContextProps } from './Context'
 import Subscription from '../utils/Subscription'
 import { useIsomorphicLayoutEffect } from '../utils/useIsomorphicLayoutEffect'
 
-function Provider({ store, context, children }) {
+function Provider({ store, context, children }: { store: any, context?: React.Context<ReduxContextProps>, children: ReactNode }) {
```

Comment:
:hand: I've always preferred defining interfaces for component props separately, right above the component definition:

```ts
interface ProviderProps {
  store: $FixTypeLater;
  // etc
}

function Provider({store, context, children}: ProviderProps) {

}
```
```